Sources/AdditiveArithmeticCartesianComponentsRepresentable.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  AdditiveArithmeticCartesianComponentsRepresentable.swift
+//  Euclid
+//
+//  Created by Ioannis Kaliakatsos on 02.12.2020.
+//  Copyright © 2020 Nick Lockwood. All rights reserved.
+//
+
+import Foundation
+
+public protocol AdditiveArithmeticCartesianComponentsRepresentable: CartesianComponentsRepresentable {}
+
+public func + <T: AdditiveArithmeticCartesianComponentsRepresentable>(lhs: T, rhs: T) -> T {
```

Comment:
Why are these public functions rather than static members of the AdditiveArithmeticCartesianComponentsRepresentable protocol?

Reply:
Legitimate remark, I will change it accordingly